src/software/ai/hl/stp/tactic/penalty_kick_tactic.h

```diff
@@ -42,23 +42,31 @@ class PenaltyKickTactic : public Tactic
     Ball getBall() const;
     Field getField() const;
 
-   private:
-    void calculateNextAction(ActionCoroutine::push_type &yield) override;
-
     // Evaluation function designed specifically for 1-on-1 penalty shots
-    bool evaluate_penalty_shot();
+    bool evaluatePenaltyShot();
+
     // Evaluation function designed specifically for determining the next potential shot
     // in a penalty kick
-    Point evaluate_next_position();
+    Point evaluateNextShotPosition();
```

Comment:
please add proper javadocs for these functions and maybe rename them to be more descriptive/specific

Reply:
javadoc-ed